doc/syntax/operators.rdoc

```diff
@@ -0,0 +1,74 @@
+= Operators
+
+In Ruby, operators such as <code>+</code>, are defined as methods on the class.
+Literals[rdoc-ref:syntax/literals.rdoc] define their methods within the lower
+level, C language. String class, for example.
+
+Ruby objects can define or overload their own implementation for most operators.
+
+Here is an example:
+
+class Foo < String
+    def &&(exp)
```

Comment:
`&&` operator can not be a method.

Reply:
!!!

I forgot to remove this when I discovered it was tokenized.

Thank you